lib/openstudio-standards/service_water_heating/information.rb

```diff
@@ -0,0 +1,81 @@
+  module ServiceWaterHeating
+    # @!group Information
+    # Methods to add get information about service water heating equipment and loops
+
+    # Determines water heater capacity and volume from an array of water use equipment.
+    #
+    # @param water_use_equipment_array [Array<OpenStudio::Model::WaterUseEquipment>] Array of water use equipment objects served by this water heater
+    # @param capacity_to_volume_ratio [Double] storage volume in gallons per kBtu/hr of capacity
+    # @param water_heater_efficiency [Double] water heater thermal efficiency, fraction
+    # @param inlet_temperature [Double] inlet cold water temperature, degrees Fahrenheit
+    # @param supply_temperature [Double] supply water temperature from the tank, degrees Fahrenheit
+    # @param peak_flow_fraction [Double] a variable for system diversity as fraction of peak flow when setting water heater capacity
+    # @param minimum_volume [Double] minimum allowable volume of the water heater, in gallons
+    # @return [Hash] hash with water_heater_capacity in watts and water_heater_volume in m^3
+    def self.water_heater_sizing_from_water_use_equipment(water_use_equipment_array,
+                                                          capacity_to_volume_ratio: 1.0,
+                                                          water_heater_efficiency: 0.8,
+                                                          inlet_temperature: 40.0,
+                                                          supply_temperature: 140.0,
+                                                          peak_flow_fraction: 1.0,
+                                                          minimum_volume: 40.0)
+      # Initialize hash
+      water_heater_sizing = {}
+
+      # Get the maximum flow rates for all pieces of water use equipment
+      adjusted_max_flow_rates_gal_per_hr = []
+      water_use_equipment_array.sort.each do |water_use_equip|
+        water_use_equip_sch = water_use_equip.flowRateFractionSchedule
+        next if water_use_equip_sch.empty?
+
+        water_use_equip_sch = water_use_equip_sch.get
+        if water_use_equip_sch.to_ScheduleRuleset.is_initialized
+          water_use_equip_sch = water_use_equip_sch.to_ScheduleRuleset.get
+          max_sch_value = OpenstudioStandards::Schedules.schedule_ruleset_get_min_max(water_use_equip_sch)['max']
+        elsif water_use_equip_sch.to_ScheduleConstant.is_initialized
+          water_use_equip_sch = water_use_equip_sch.to_ScheduleConstant.get
+          max_sch_value = OpenstudioStandards::Schedules.schedule_constant_get_min_max(water_use_equip_sch)['max']
+        elsif water_use_equip_sch.to_ScheduleCompact.is_initialized
+          water_use_equip_sch = water_use_equip_sch.to_ScheduleCompact.get
+          max_sch_value = OpenstudioStandards::Schedules.schedule_compact_get_min_max(water_use_equip_sch)['max']
```

Comment:
why not just [OpenstudioStandards::Schedules.schedule_get_min_max(water_use_equip_sch)['max']](https://github.com/NREL/openstudio-standards/blob/e5ef38ac30dcbc1a3eefa3901fa3438588452f5a/lib/openstudio-standards/schedules/information.rb#L15)

Reply:
that is a better choice. changed.